src/helpers.js

```diff
@@ -22,6 +22,16 @@ const getDirFromFilePath = (filePath: FilePath): FilePath => path.parse(filePath
 const getNearestEslintignorePath = (filePath: FilePath): ?FilePath =>
   findCached(getDirFromFilePath(filePath), '.eslintignore');
 
+const getLocalPrettierPath = (filePath: ?FilePath): ?FilePath => {
+  if (!filePath) return null;
+
+  const indexPath = path.join('node_modules', 'prettier', 'index.js');
+  const dirPath = getDirFromFilePath(filePath);
+  if (!dirPath) return null;
```

Comment:
Yep, makes sense, ternary it is. The reason to split the path/lookup logic from the actual 
requiring is it makes it easier to test in isolation.